src/popup.js

```diff
@@ -205,7 +203,7 @@ function _badgerStatusTitle(action){
     block:        status_block,
     cookieblock:  status_cookieblock,
     noaction:     status_noaction
-  }
+  };
```

Comment:
It was there for DRYness before the i18n changes. Doesn't make sense any more. I will refactor this function.